src/main/java/org/littil/api/auth/TokenHelper.java

```diff
@@ -35,12 +35,16 @@ public <T> T getCustomClaim(String claimName) {
         return accessToken.getClaim(claimNamespace.concat(claimName));
     }
 
-    public UUID getCurrentUserId() {
+    public Optional<UUID> currentUserId() {
```

Comment:
dit is omdat de ApplicationLifeCycle geen user-context heeft voor bij aanmaken van de dev-users, schools and guest-teachers. Misschien dat we dit anders moeten oplossen (b.v. deze context zelf setten vanuit ApplicationLifeCycle)

Reply:
ik denk dat het mooier is om AuditableEntityListener uit te breiden met een setter om een custom UserId te setten en dat dan vanuit ApplicationLifeCycle te doen

Reply:
Jouw optionele implementatie is ok voor nu.